luigi_pipeline/lib/model/base_mt_schema.py

```diff
@@ -1,8 +1,7 @@
+from collections import defaultdict
```

Comment:
There's a configuration option (`force_sort_within_sections`) that enables full alphabetical rather than splitting the `import x` imports from the `from an import b` imports.  With the option disabled (the default), this block would be:
```
import logging
from collections import defaultdict
from inspect import getmembers
from typing import List
```

rather than what it is here.  

I'm extremely unopinionated about which option is better, though I was slightly surprised by the default!  Does anyone else have thoughts?

Reply:
I didn't know much about the `isort` before. Both options look good to me.

Reply:
👍 Slight preference > no preference!  Also... I was looking at a block elsewhere that was:

```
import a
from b import c
import d
from e import f
import h
```

and thought it looked a bit gross.  So I think I agree!